app_dart/bin/server.dart

```diff
@@ -216,6 +216,33 @@ Future<void> main() async {
         delegate: GetStatus(config),
       ),
 
+      /// Returns a list of the commit shas that had green runs.
```

Comment:
This doc should be in the class.

See https://github.com/flutter/flutter/issues/97675